llvm/tools/llvm-objdump/llvm-objdump.cpp

```diff
@@ -1486,7 +1486,9 @@ collectLocalBranchTargets(ArrayRef<uint8_t> Bytes, MCInstrAnalysis *MIA,
                           std::unordered_map<uint64_t, std::string> &Labels) {
   // So far only supports PowerPC and X86.
```

Comment:
The comment is now stale. I suggest that we replace it with `// Supported by certain targets.` so that it will not become stale whenever we add a new target.

Reply:
Oh, my bad, will update.